src/arkode/arkode_relaxation.c

```diff
@@ -39,14 +39,15 @@ static int arkRelaxAccessMem(void* arkode_mem, const char* fname,
 {
   if (!arkode_mem)
   {
-    arkProcessError(NULL, ARK_MEM_NULL, "ARKODE", fname, MSG_ARK_NO_MEM);
```

Comment:
The input `fname` is no longer used and could be removed.

Reply:
This line youre referring to was a line that was removed.